bindgen/src/bindings/cpp/templates/wrapper.hpp

```diff
@@ -42,6 +61,19 @@ struct {{ name }};
 struct {{ name }};
 {%- when Type::CallbackInterface { module_path, name } %}
 struct {{ name }};
+{%- when Type::Custom { module_path, name, builtin } %}
+{%- match config.custom_types.get(name.as_str()) %}
+{%- when None %}
+typedef {{ builtin|type_name }} {{ name }};
+{%- when Some with (type_config) %}
+{%- match type_config.type_name %}
+{%- when Some with (type_name) %}
+typedef {{ type_name }} {{ name }};
+{%- let type_name = typ|type_name %}
```

Comment:
Does `type_name` and `class_name` actually do anything here?

Reply:
nice catch! left it by accident when moving code around, removed